### PR TITLE
Update to Dart 2.13

### PR DESCRIPTION
**Pull request by robbecker-wf**
## Motivation
  
We want to update everything at Workvia to work with Dart 2.13

## Changes
  
Update dependencies and Dockerfile to work under Dart 2.13
Removes last use of dart2_constant

## Review
_[See CONTRIBUTING.md][contributing-review-types] for more details on review types (+1 / QA +1 / +10) and code review process._

  

Please review: 

### QA Checklist
- [ ] Tests were updated and provide good coverage of the changeset and other affected code
- [ ] Manual testing was performed if needed
    - [ ] Steps from PR author: 
        
    - [ ] Anything falling under manual testing criteria [outlined in CONTRIBUTING.md][contributing-manual-testing]

## Merge Checklist
While we perform many automated checks before auto-merging, some manual checks are needed:
- [ ] A Client Platform member has reviewed these changes
- [ ] There are no unaddressed comments _- this check can be automated if reviewers use the "Request Changes" feature_
- [ ] _For release PRs -_ Version metadata in Rosie comment is correct

[contributing-review-types]: https://github.com/Workiva/abide/blob/master/CONTRIBUTING.md#review-types
[contributing-manual-testing]: https://github.com/Workiva/abide/blob/master/CONTRIBUTING.md#manual-testing-criteria

### CODE REVIEW

**Comment by aviary2-wf** [comment]
## Security Insights
No security relevant content was detected by automated scans.

## Action Items
* Review PR for [security impact](https://wiki.atl.workiva.net/display/SECURITY/Development+Security+Review+Guidelines); comment "security review required" if needed or unsure
* Verify [`aviary.yaml`](https://wiki.atl.workiva.net/pages/viewpage.action?pageId=42762752#SecurityInsightFramework&Tooling-ConfiguringAviary) coverage of security relevant code
---
_Questions or Comments? Reach out on Slack: #support-infosec._

**Comment by robbecker-wf** [comment]
@rmconsole-wf how you doing? You've been catatonic for hours.
image

**Comment by robbecker-wf** [comment]
Comment to restart Rosie

File: Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM google/dart:2.7.2 as build
+FROM google/dart:2.13.4 as build
```
Comment (robbecker-wf) [inline Dockerfile:1]: Using the google image for open source projects

**Review by evanweible-wf — APPROVED** [review]

**Comment by robbecker-wf** [comment]
@Workiva/release-management-p

**Review by rmconsole-wf — APPROVED** [review]
+1 from RM

**Pull request merged**